src/Core/Datatypes/ColorMap.cc

```diff
@@ -153,7 +154,7 @@ double ColorMap::getTransformedValue(double f) const
   static bool x = true;
```

Comment:
This weird nonsense code was put in a long time ago due to a strange crash which I can't even remember. Let's try removing the whole block

Reply:
https://github.com/SCIInstitute/SCIRun/commit/cf0fb5aa4a5c438e96e3a42545cfec8e913ae5ed#diff-e6d3c5861036ceaf0b1ed068fc84a547

Reply:
I removed the code and it built and ran just fine on mac and windows